### ISSUE TITLE
Unable to signup into CSSBattle

### ISSUE THREAD

**Issue by PintuPx** [cssbattle/forum#99]
I am not able to signup for the CSS battle .
I tried the link - https://cssbattle.dev/login and choose the first option "Sign in with Email". After providing the email I got a notification saying "A password-less login link has been sent to your email". And I tried the link, the response was a blank page and in the chrome dev tool there was an error 
```
GET https://cssbattleapp.firebaseapp.com/__/auth/action.js net::ERR_EMPTY_RESPONSE
action?apiKey=AIzaSyCR1mcn0jOf7IXUmuo&mode=signIn&oobCode=JDp75pWbta_6mibw5ePXeKYcp7iHdZMvRUP1Ghw8fBoAAAF7xD4nPg&continueUrl=https://cssbattle.dev/login&lang=en:9 Uncaught ReferenceError: fireauth is not defined
    at action?apiKey=AIzaSyCR1oteCf0oGnq0HXymDmcn0jOf7Imuo&mode=signIn&oobCode=JDp75pWbta_6mibw5ePXeKYcp7iHdZMvRUP1Ghw8fBoAAAF7xD4nPg&continueUrl=https://cssbattle.dev/login&lang=en:9
```

And also I tried to log in with Google, a popup is coming and closing after some time no further action is taking place.

**Comment by chinchang** [comment]
Can you please check if 3rd party cookies are enabled in your browser or not? Which browser you using?

**Comment by bhaveshxrawat** [comment]
the exact same thing is happening with me. I'm on Chrome Browser. On my smartphone, it works just fine but is giving me a hard time on my Mac.

@PintuPx did you manage to solve it?

**Comment by PintuPx** [comment]
No, Still I am unable to log in, I am using Firefox version 98.0.2 (64 bit) on windows 10

**Comment by chinchang** [comment]
Hey @PintuPx apologies for lack of followup here. Were you able to login or does the problem still persists?
